src/benchmarker.js

```diff
@@ -156,8 +170,8 @@ function dump(
   let code = serialized.code;
   let total = Date.now() - start;
 
-  if (code.length >= 1000 || outputFilename) {
-    let filename = outputFilename || name + "-processed.js";
+  if (code.length >= 1000 || typeof outputFilename !== "undefined") {
```

Comment:
instead of "typeof outputFilename !== "undefined"" just write "outputFilename !== undefined".